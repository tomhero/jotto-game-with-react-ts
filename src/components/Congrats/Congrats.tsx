import React from 'react'

export type CongratsProps = {
};

const Congrats = (props: CongratsProps) => {
  return (
    <div data-test="component-congrats">

    </div>
  )
}

export default Congrats;
