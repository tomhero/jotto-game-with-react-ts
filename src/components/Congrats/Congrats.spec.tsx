import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import Congrats, { CongratsProps } from './Congrats';
import { findByTesttAttr } from '../../../test/test-utils';

describe('<Congrats />', () => {
  let congratsWrapper: ShallowWrapper;
  let defaultProps: CongratsProps = {
  };

  beforeEach(() => {
    congratsWrapper = shallow(<Congrats {...defaultProps} />);
  });

  it('renders the component', () => {
    expect(congratsWrapper).toMatchSnapshot();
    const containerComponent = findByTesttAttr(congratsWrapper, 'component-congrats');
    expect(containerComponent).toHaveLength(1);
  });

})

