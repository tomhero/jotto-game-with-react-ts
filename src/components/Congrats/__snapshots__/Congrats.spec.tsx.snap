// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Congrats /> renders the component 1`] = `
<div
  data-test="component-congrats"
/>
`;
